lib/tag-handler.js

```diff
@@ -26,7 +26,7 @@ const versionPattern = /([0-9]\d*\.[0-9]\d*\.[0-9]\d*)([-]\w+)?([.]\d+)?/
 module.exports.tagHandler = (command) => async ({ github, verbose, number }) => {
   const _exec = exec({ verbose })
   try {
-    const { version } = JSON.parse(fs.readFileSync(`${__dirname}/../package.json`, 'utf8'))
+    const { version } = JSON.parse(fs.readFileSync(`${process.cwd()}/package.json`, 'utf8'))
```

Comment:
lol you're right. `__dirname` is respective to `@eqworks/release`, not the target project repository 🤦